ngraph/core/include/ngraph/pass/itt.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/cc/selective_build.h>
+#include <openvino/itt.hpp>
```

Comment:
these headers are NOT a part of release package

Reply:
Yeah right. I will think how to enable this file only for open source.

Reply:
looks like we need to move all this stuff to `openvino/itt`